### PR TITLE
feat(bulk-import): add repo row actions

### PR DESCRIPTION
**Pull request by ciiay**
Resolves:
[RHDIP-1769](https://issues.redhat.com/browse/RHIDP-1769): Add row actions to the Added repositories table

**Note**:
This PR is based off #1541 .
Screen recording(updated 6/4):

https://github.com/janus-idp/backstage-plugins/assets/26255262/ef1e7d49-ffad-4b5b-92a0-93422bd090e3

Updates(5/1):
rhidp_1769_update_51_2
rhidp_1769_update_51_3
rhidp_1769_update_51_4

Update(6/4)
image

### CODE REVIEW

**Comment by ShiranHi** [comment]
@ciiay l have some adjustments for the tooltips we show and for the modal:
1. When we show the external link icon, the tooltip test should be "View catalog-info.yaml file"
image01

2. Let's have "Remove" for this trash icon
image02

3. For the last icon let' use just "Refresh"
image03

About the modal, the content seems not aligned ([Figma](https://www.figma.com/design/09Iltd57RiuqOtKWkjy8MH/Bulk-import?node-id=473%3A668&t=4dPFf2otdEAdEmK8-1)), and the text should be "Removing a repository erases all associated information from the Catalog page."
image04

**Comment by ciiay** [comment]
Hi @ShiranHi , did you take these screenshots from the screen recording video? I didn't update the screen recording, only uploaded some updated screenshots which show the changes I made to address your previous comments about tooltip message and alert model alignment. So the screen recording didn't reflect the code change.

I just uploaded a new screen recording. Let me know if it looks good or not now.

**Comment by ShiranHi** [comment]
> I just uploaded a new screen recording. Let me know if it looks good or not now.

Yes, I took it from the screen recording, thank you for updating it. 
It seems that the tooltip text is correct, but the text "Removing a repository..." is not aligned, could you please take a look on that?

**Comment by ciiay** [comment]
Hi @ShiranHi , thanks for the pointer. I over looked the text alignment in my previous fix. Uploaded a new screen recording and a screenshot to reflect the fix.

File: plugins/bulk-import/src/components/PreviewFile/PreviewPullRequest.tsx [hunk 1]
```diff
@@ -12,13 +12,13 @@ import {
...
 
 import { makeStyles } from '@material-ui/core';
-import TextField from '@material-ui/core/TextField';
 import Autocomplete from '@mui/material/Autocomplete';
 import Box from '@mui/material/Box';
 import Checkbox from '@mui/material/Checkbox';
 import CircularProgress from '@mui/material/CircularProgress';
 import FormControlLabel from '@mui/material/FormControlLabel';
 import FormHelperText from '@mui/material/FormHelperText';
+import TextField from '@mui/material/TextField';
```
Comment (debsmita1) [inline plugins/bulk-import/src/components/PreviewFile/PreviewPullRequest.tsx:21]: The MUI4 text field was used to resolve the issue that occurred when resizing the screen while the preview side panel was open.

Screenshot 2024-06-25 at 5 23 01 PM

**Comment by debsmita1** [comment]
@ciiay  Changes made to the preview file in the org view no longer persist after saving.

**Comment by debsmita1** [comment]
Verified the changes
/lgtm

Thank you @ciiay !

**Pull request merged**